main.java

```diff
@@ -1 +1,50 @@
+import java.util.*;
+class stdmarks{
 
+              public static void main (String args[]){
+              
+              
+                  System.out.println("enter the name of the student");        
+                  Scanner sc =new Scanner(System.in);        
+                  String name =sc.nextLine();
+                  System.out.println("enter the section of the student");
+                  char section=sc.next().charAt(0);
+                  System.out.println("enter roll no of the student");
+                  Scanner input=new Scanner(System.in);
+                  String roll=input.nextLine();
+                  System.out.println("enter the age of the student");
+                  int age=sc.nextInt();
+                  int marks[];
+                  int t=0;
+                  marks=new int[10];
+                  /*marks[0]=mathematics,marks[1]=physics;marks[2]=chemistry;marks[3]=english,marks[4]=biology;marks[5]=environmental sciences,*/
```

Comment:
Nice and thoughtful to mention the order of subjects in the array "marks". 
It adds to the "ease in debuggability".